static/app/components/eventOrGroupHeader.tsx

```diff
@@ -116,6 +118,7 @@ function EventOrGroupHeader({
               eventID ? `events/${eventID}/` : ''
             }`,
             query: {
+              referrer: source || 'event-or-group-header',
```

Comment:
can this source be an empty `string` or `0`? ca we maybe only use `??` here?

Reply:
@priscilawebdev If someone put in an empty string or a 0, then defaulting to `'event-or-group-header` would be better